Plugin/UIScopeLabel.php

```diff
@@ -41,20 +44,22 @@ public function afterSetupAttributeMeta(EavModifier $subject, $result, ProductAt
             return $result;
         }
 
+        if ($attribute->getFrontendInput() === AttributeInterface::FRONTEND_INPUT) {
```

Comment:
Can you explain why you put this block here and not after the next five lines? Putting it here changes the logic, and I'm not sure at the moment whether this has an impact or not.

Reply:
@avstudnitz Hello, the logic was not changed. I just move the condition at the top and use early exit. 
![image](https://github.com/user-attachments/assets/edda83e3-8aa4-4b5d-bb0f-0e47a07ddbbb)

When you will check option "Hide whitespace" everything should be clear.